styles/minimap-codeglance.atom-text-editor.less

```diff
@@ -23,14 +23,12 @@ minimap-codeglance {
     width: 100%;
     height: 100%;
 
-    &::shadow {
```

Comment:
I don't think it really matters _too much_ in this case, but for consistency, I think you're right; `&.editor` would be correct.  I'm no expert on Atom CSS stylings.